src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.tsx

```diff
@@ -81,6 +81,18 @@ function makeImageElement({ noBorder }: { noBorder: ImageProps['noBorder'] }) {
   )
 }
 
+/**
+ * This component overrides the custom 'li' running in the @hashicorp/remark-plugins
+ * "typography" plugin. This plugin is run via the DocsPage component in @hashicorp/docs-page
+ * so we need to override it here. https://github.com/hashicorp/web-platform-packages/blob/main/packages/remark-plugins/plugins/typography/index.js#L13
+ *
+ * @TODO cleanup use of remark plugins in @hashicorp/docs-page so these overrides aren't necessary.
+ * Or decouple the dev portal docs page from this legacy component. https://app.asana.com/0/1202097197789424/1202310153805071
+ */
+function ListItem({ children }: { children: React.ReactNode }) {
+  return <li>{children}</li>
+}
```

Comment:
[See follow up ticket](https://app.asana.com/0/1202097197789424/1202310153805071) here for full context: 
> Currently, docs content is process through the [`DocsPage` component](https://github.com/hashicorp/react-components/tree/main/packages/docs-page) where the [default markdown plugins](https://github.com/hashicorp/react-components/blob/main/packages/docs-page/render-page-mdx.ts#L30-L33) are applied. In these defaults, [the typography plugin](https://github.com/hashicorp/web-platform-packages/blob/main/packages/remark-plugins/plugins/typography/index.js) is ran ([see here](https://github.com/hashicorp/web-platform-packages/blob/c961e2cb9268712cfd8f06ccba51fe07252c1935/packages/markdown-utils/index.ts#L1)).

> This typography plugin uses legacy `g-type-*` classnames to apply type styles that no longer abide by the design system used in dev dot. Type styles now are applied within the dev dot application. 

> We need to either break dev dot from using this docs page component or upgrade it via a canary version that adjusts this remark plugin from being run.

Reply:
🙌 Task links in TODO comments 💯

Reply:
This is great context, thanks for linking in the code! 👏🏻